docs/api-docs/storefront/graphql/graphql-api-overview.mdx

```diff
@@ -222,6 +226,9 @@ mutation Login($email: String!, $pass: String!) {
   }
 }
 ```
+<Callout type="info">
+NOTE: Customers are restricted from multiple logins from different devices. 
```

Comment:
```suggestion
    Note: A customer cannot be signed in to their account on multiple devices at the same time. 
```

Reply:
can we say more about the default behavior here? does BC automatically sign them out of the old device when you run the customer login mutation for a session on a new device?